Lib/fontTools/t1Lib/__init__.py

```diff
@@ -367,3 +513,72 @@ def stringToLong(s):
 	for i in range(4):
 		l += byteord(s[i]) << (i * 8)
 	return l
+
+
+# PS stream helpers
+
+font_dictionary_keys = copy.copy(_type1_pre_eexec_order)
+# t1write.c:writeRegNameKeyedFont
+# always counts following keys
+font_dictionary_keys.remove("FontMatrix")
+
+FontInfo_dictionary_keys = copy.copy(_type1_fontinfo_order)
+# extend because AFDKO tx may use following keys
+FontInfo_dictionary_keys.extend([
+	"FSType",
+	"Copyright",
+])
+
+Praivate_dictionary_keys = [
+	# We don't know what names will be actually used.
+	# "RD",
+	# "ND",
+	# "NP",
+	"Subrs",
+	"OtherSubrs",
+	"UniqueID",
+	"BlueValues",
+	"OtherBlues",
+	"FamilyBlues",
+	"FamilyOtherBlues",
+	"BlueScale",
+	"BlueShift",
+	"BlueFuzz",
+	"StdHW",
+	"StdVW",
+	"StemSnapH",
+	"StemSnapV",
+	"ForceBold",
+	"LanguageGroup",
+	"password",
+	"lenIV",
+	"MinFeature",
+	"RndStemUp",
+]
+
+# t1write_hintothers.h
+hintothers = \
```

Comment:
This may look nicer as a triple-quoted string.

Reply:
I'll use a triple-quoted string.